roles/lib_openshift/src/class/oc_storageclass.py

```diff
@@ -0,0 +1,147 @@
+# pylint: skip-file
+# flake8: noqa
+
+# pylint: disable=too-many-instance-attributes
+class OCStorageClass(OpenShiftCLI):
+    ''' Class to wrap the oc command line tools '''
+    kind = 'sc'
```

Comment:
3.5 does not have this short form, had to change it to storageclass so the oc commands work